--- origsrc/gst-plugins-gl/docs/libs/Makefile.am	2009-02-03 22:20:00.000000000 -0600
+++ src/gst-plugins-gl/docs/libs/Makefile.am	2009-02-03 23:30:56.691250000 -0600
@@ -83,7 +83,7 @@ extra_files =
 # CFLAGS and LDFLAGS for compiling scan program. Only needed if your app/lib
 # contains GtkObjects/GObjects and you want to document signals and properties.
 GTKDOC_CFLAGS = $(GST_PLUGINS_BASE_CFLAGS) $(GST_BASE_CFLAGS) \
-	-I$(top_builddir)/gst-libs/gst/gl
+	-I$(top_srcdir)/gst-libs/gst/gl
 GTKDOC_LIBS = $(SCANOBJ_DEPS) $(GST_BASE_LIBS) \
 	$(top_builddir)/gst-libs/gst/gl/libgstgl-@GST_MAJORMINOR@.la
 
--- origsrc/gst-plugins-gl/docs/plugins/inspect-build.stamp	1969-12-31 18:00:00.000000000 -0600
+++ src/gst-plugins-gl/docs/plugins/inspect-build.stamp	2009-02-04 00:57:36.910000000 -0600
@@ -0,0 +1 @@
+timestamp
\ No newline at end of file
--- origsrc/gst-plugins-gl/docs/plugins/inspect.stamp	1969-12-31 18:00:00.000000000 -0600
+++ src/gst-plugins-gl/docs/plugins/inspect.stamp	2009-02-04 00:57:41.566250000 -0600
@@ -0,0 +1 @@
+timestamp
\ No newline at end of file
